File: electron-admin/src/main/java/cn/bestrivenlf/electronadmin/entity/Chateach.java
package cn.bestrivenlf.electronadmin.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "t_chateach")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Chateach {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column
    private String ip1;

    @Column
    private String userName1;

    @Column
    private String ip2;

    @Column
    private String userName2;

    @Column
    private String fromIdentity1;

    @Column
    private String fromIdentity2;


    @Column
    private String content;

    @Column
    private Long gmtCreate;

    @Column
    private Long gmtModified;

}

File: front/src/renderer/api/chateach.js
import request from '@/utils/request'

export function getChatEachList(ip1,ip2) {
  return request({
    url: `/chateach/getAllChat/${ip1}/${ip2}`,
    method: 'get'
  })
}

export function addChatEach(userName1, ip1, userName2, ip2, content,fromIdentity1,fromIdentity2) {
  return request({
    url: '/chateach/addOneChat',
    method: 'post',
    data: {
      userName1,
      ip1,
      userName2,
      ip2,
      content: content,
      fromIdentity1,
      fromIdentity2
      
    }
  })
}
